lte/gateway/c/core/oai/tasks/amf/amf_app_state_manager.cpp

```diff
@@ -61,14 +61,20 @@ void clear_amf_nas_state() {
   AmfNasStateManager::getInstance().free_state();
 }
 
-map_uint64_ue_context_t get_amf_ue_state() {
+map_uint64_ue_context_t* get_amf_ue_state() {
   return AmfNasStateManager::getInstance().get_ue_state_map();
 }
 
 void delete_amf_ue_state(imsi64_t imsi64) {
-  auto imsi_str =
-      magma5g::AmfNasStateManager::getInstance().get_imsi_str(imsi64);
-  magma5g::AmfNasStateManager::getInstance().clear_ue_state_db(imsi_str);
+  OAILOG_DEBUG(LOG_MME_APP, "Delete AMF ue state, %lu", imsi64);
+#if !MME_UNIT_TEST
+  auto imsi_str = AmfNasStateManager::getInstance().get_imsi_str(imsi64);
```

Comment:
Please add comment here explaining that the removal is happening from the DB or from AMF client servicer

Reply:
Comment Addressed